config/routes.rb

```diff
@@ -59,6 +59,14 @@
         namespace :example do
           get 'iterations_summary_table/:id', to: 'iterations_summary_table#index', as: 'iterations_summary_table'
         end
+        namespace :notifications do
+          resource :icon, only: %i[show update]
+        end
+        namespace :mentoring do
+          resource :mentor_conversations_list, only: [:show] do
+            resources :conversations, only: [:index], controller: "mentor_conversations_lists/conversations"
```

Comment:
Can you merge these two controllers and have an extra action on the one please. I don't want to have two controllers, each with one action, for something here - it makes it harder to work out what's happening, and I don't mind about keeping this "Rails Restful". 

So:
```
resource :mentor_conversations_list, only: [:show] do
  get :conversations, on: :collection
end
```

and move the conversations action onto the `MentorConversationsListController`.

Reply:
What makes it harder to work out?

Reply:
Quite simply, I have to look in two files to see what's going on, rather than keeping all the things in one file together.

Reply:
I basically have a strong preference against having lots of controllers with relatively shared/overlapping functionality. It makes it much harder to get a feel for what is going on when you come to the code cold, than if you can look in one place where all the code is together. I would rather encapsulate the functionality of all the relevant things in one controller, than having multiple controllers for the conformity with the REST-ideals.

I'll add this to the standards document.